src/common/Input.js

```diff
@@ -8,10 +8,10 @@ import type { ThemeData } from '../styles';
 import { ThemeContext, HALF_COLOR, BORDER_COLOR } from '../styles';
 
 export type Props = $ReadOnly<{|
-  // TextInput's definition changes across the RN v0.61 -> v0.62
-  // upgrade; we'll handle that change after the upgrade.
+  // See point 2 at
+  // https://github.com/facebook/react-native/issues/28459#issuecomment-609957836
   // $FlowFixMe
```

Comment:
It takes a bit of unpacking to see from this that we expect this to go away with RN v0.63.

How about mentioning this in a comment on #4245, and then pointing there?